src/components/Environment/environment-actions.ts

```diff
@@ -18,5 +19,15 @@ export const useEnvironmentActions = (environment: EnvironmentKind): Action[] =>
       id: `delete-${environment.metadata.name.toLowerCase()}`,
       label: 'Delete',
     },
+    {
+      cta: () =>
+        showModal(
+          createEditStrategyModal({
+            obj: environment,
+          }),
+        ),
+      id: `edit-strategy-${environment.metadata.name.toLowerCase()}`,
+      label: 'Edit deployment strategy',
+    },
```

Comment:
this action should be before delete action.

Reply:
Updated.